program/tests/namespace_royalty_enforcement.rs

```diff
@@ -0,0 +1,1133 @@
+#![cfg(feature = "test-bpf")]
+
+pub mod utils;
+
+use std::collections::HashSet;
+
+use mpl_token_auth_rules::{
+    error::RuleSetError,
+    instruction::{builders::ValidateBuilder, InstructionBuilder, ValidateArgs},
+    payload::{Payload, PayloadType},
+    state::{CompareOp, Rule, RuleSetV1},
+};
+use solana_program::{instruction::InstructionError, pubkey, pubkey::Pubkey};
+use solana_program_test::{tokio, ProgramTestContext};
+use solana_sdk::{
+    instruction::AccountMeta,
+    signature::Signer,
+    signer::keypair::Keypair,
+    system_instruction,
+    transaction::{Transaction, TransactionError},
+};
+use utils::{
+    create_associated_token_account, create_mint, program_test, DelegateScenario,
+    MetadataDelegateRole, Operation, PayloadKey, TokenDelegateRole, TransferScenario,
+};
+
+const ADDITIONAL_COMPUTE: u32 = 1_400_000;
```

Comment:
What does additional compute need to be if there's not 70 pubkeys?  Like what does 20 pubkeys need for compute for creation?

Reply:
450Kish